src/main/resources/uk/ac/ebi/embl/api/validation/data/feature-qualifier-rename.tsv

```diff
@@ -2,3 +2,4 @@
 
 mobile_element	mobile_element_type
 label	note
+geo_loc_name	country
```

Comment:
the country should be changed to geo_loc_name and country qualifier related validation/fixes should be changed to use geo_loc_name instead.